content/blog/eclipse-temurin-slsa2/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: SLSA level 2 compliance for Eclipse Temurin
+date: "2022-11-11T12:00:00+00:00"
+author: sxa
+description: Eclipse Temurin by Adoptium is compliant with level 2 of the SLSA secure development framework.
+tags:
+  - Temurin
+  - Security
+---
+
+# Introduction to SLSA
+
+[SLSA](https://slsa.dev) is a framework with inndividual levels to indicate
+increasing levels of security which software products can work towards. 
+Today, we are pleased to announce that the Eclipse Temurin build and
+distribution process is compliant with Level 2 of the SLSA specification.
+
+What does this mean in practice? The [SLSA v0.1 levels are defined at this
+link](http://slsa.dev/spec/v0.1/levels) and this means we are compliant with
+levels 1 and 2 of the specification.
+
+## SLSA Level 1
+
+Level 1 means that the build process is full scripted and automated and we
+are now producing Software Bill Of Material (SBOM) documents describing the
+items which went into producing each of our builds which are published
+alongside the downloads.  We have been compliant with level 1 for some time:
+The level 1 requirements are:
+
+- **Build - scripted build**: Our build steps are defined completely by the
+  jenkins pipelines from the [ci-jenkins-pipelines](https://github.com/adoptium/ci-jenkins-pipelines) repository and the
+  underlying scripts run on the build machines from the
+  [temurin-build](https://github.com/adoptium/temurin-build)
+  repository - starting at
+  [make-adopt-build-farm.sh](https://github.com/adoptium/temurin-build/blob/master/build-farm/make-adopt-build-farm.sh)
+
+- **Provenance - available**: We create an SBOM in OWASP CycloneDX format
+  along with our releases which contains all of the information about how
+  the builds are produced, which should allow someone to rebuild if required
+  for any reason.  This includes the full set of parameters which we use to
+  the makejdk_any_platform.sh script (invoked from
+  make-adopt-build-farm.sh), the source repository tags (`scmRef`) which was
+  used by the build process to produce the builds, the output from the
+  openjdk `configure` invocation and various other pieces of information. 
+  We are continually evolving the specific details which we include in the
+  SBOM.  If you want to join in the discussion on the content, you can find
+  the conversation in
+  [temurin-build#3013](https://github.com/adoptium/temurin-build/issues/3013)
```

Comment:
Or join the conversation in our Slack workspace in the secure-dev channel

Reply:
Added a line to the same effect.